lib/omise/res/OmiseApiResource.php

```diff
@@ -199,11 +199,16 @@ private function _executeCurl($url, $requestMethod, $key, $params = null)
      */
     private function _executeTest($url, $requestMethod, $key, $params = null)
     {
-        // Remove Http, Https protocal from $url (string).
-        $request_url = preg_replace('#^(http|https)://#', '', $url);
-
         // Remove slash if it had in last letter.
```

Comment:
@iwat a last small point here, since we removed the previous implementation, I think this comment is not reflect to the current implementation much, shall we remove? 

LGTM for the rest 👍